python/pyspark/sql/tests.py

```diff
@@ -3308,12 +3306,12 @@ def test_vectorized_udf_invalid_length(self):
         from pyspark.sql.functions import pandas_udf, col
         import pandas as pd
         df = self.spark.range(10)
-        raise_exception = pandas_udf(lambda: pd.Series(1), LongType())
+        raise_exception = pandas_udf(lambda i: pd.Series(1), LongType())
         with QuietTest(self.sc):
             with self.assertRaisesRegexp(
                     Exception,
```

Comment:
Here too, while we are here, let's catch narrower exception type. Looks `RuntimeError`.

Reply:
This one is actually a `Py4jJavaError` so I don't think we can recognize the `RuntimeError`